nodebuilder/state/config.go

```diff
@@ -2,25 +2,21 @@ package state
 
 import (
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
-
-	"github.com/celestiaorg/celestia-node/state"
 )
 
 var defaultKeyringBackend = keyring.BackendTest
 
 // Config contains configuration parameters for constructing
 // the node's keyring signer.
 type Config struct {
-	KeyringAccName string
-	KeyringBackend string
-	GranterAddress state.AccAddress
+	KeyringKeyNames []string
```

Comment:
What's the goal of this? Is that a filter specifying a list of keys the API can use? Or what would be the reason for the user to add new key names here?

Reply:
From the code it looks like we get the first one and ignore all the others. What's the idea?

Reply:
`// go through all keys in the config and check their availability in the KeyStore.`

Reply:
I realized that we don't need a slice of keys. KeyringAccName should only specify the default signer. Will return everything back. Thanks